File: Hello-everyone-/Java_Projects/NumZero.java
public class NumZero
   {
      public static void main(String[] args)
      {	 double[] list = {0.0,8.78,0.0,0.0,0.0,1.1,0.0,58.7,0.5,0.0,100.45,0.25,0.0,5.5,0.0}; 
   	 int i;             
	 int zero=0;         
               for ( i = 0 ; i < list.length ; i++ )
   	 {
   	    if ( list[i] == 0.0 )
   	    ++zero;
   	 }
       	 System.out.println( zero );   
          
      }
   }

File: Hello-everyone-/Python_Projects/NumZero.py

def number_of_zeros(lst):
    count=0
    for x in list:
       if x==0:
         count+=1
    return count
          
list=[1,0,2,0,5,0,8,0,9,0,6,4,7,5,0,0,0,0,7,8,5,10]
answer = number_of_zeros(list)

print("you have these many zero's",answer)
